pkgs/build-support/composer/fetchcomposer.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, composer }:
```

Comment:
I am not sure about this. Is composer actually deterministic these days? Last time I tried the order of items in the lock file changed based on composer version.

Reply:
Unfortunately @Ma27, Snipe IT's release tarballs do not contain the vendor directory.

@jtojnar, I've checked and composer is not deterministic - I'm not sure how to proceed with packaging this as a result.